packages/nocodb/src/models/PresignedUrl.ts

```diff
@@ -88,13 +88,17 @@ export default class PresignedUrl {
 
   public static async getSignedUrl(
     param: {
-      path: string;
+      pathOrUrl: string;
       expireSeconds?: number;
       filename?: string;
     },
     ncMeta = Noco.ncMeta,
   ) {
-    let path = param.path.replace(/^\/+/, '');
+    const isUrl = param.pathOrUrl.startsWith('http');
```

Comment:
@mertmit Shouldn't this be a bit more precise and check for the whole *scheme* case-insensitively, i.e. using a regex like

```js
const isUrl = /^https?:\/\//i.test(param.pathOrUrl);
```

Reply:
Thanks for suggestion I've applied it